tsconfig.json

```diff
@@ -17,5 +17,8 @@
     "strictBindCallApply": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true
+  },
```

Comment:
how about this watchOptions ?

Reply:
This is config for development our app in Docker environment, it will help watch files change